mimic/apache.go

```diff
@@ -0,0 +1,320 @@
+package mimic
+
+import (
+	"bytes"
+	"crypto/rand"
+	"fmt"
+	"net"
+	"net/http"
+	"text/template"
+	"time"
+)
+
+type apacheMimic struct {
+	conn net.Conn
+	req  *http.Request
+	path string
+}
+
+var (
+	Host         string
+	Port         string
+	lastModified = time.Now().Format("Fri, 22 Oct 2015 11:52:25 GMT")
```

Comment:
Weird things happen on my go1.5.1, the formatted string becomes something like `Fri, 3030 Oct 301052 1010:5230:3052 GMT`, hope it's a new introduced bug and not in previous versions.